packages/cli/dist/declarations/src/index.d.ts

```diff
@@ -1 +1 @@
-export {};
+import "dotenv/config";
```

Comment:
bad merge or something?

Reply:
Nah i just hadn't run `build` in my branch i think.

Thinking of `.gitignore`ing dist now that we don't need it for dev... it creates a bunch of noise we don't need.

Reply:
let's do it